### DISCUSSION TITLE
Is the core project abandoned?

### DISCUSSION THREAD

**Discussion by miloslavnosek** [ScoopInstaller/Scoop#5802] [General]
So software in buckets is generally updated frequently, which is great, since that way scoop still works great and makes managing software on Windows so much easier and pleasant. It's like a whole new world.
The main repo though hasn't been updated for well over a year. Is that cause for concern? Is scoop itself abandoned while only buckets get updated?

**Comment by rashil2000** [comment]
The core repo has been updated, the work happens in the `develop` branch. 

But yes, Scoop does suffer from the lack of active maintainers - we're not able to devote our free time here as much as we'd like.

**Reply by miloslavnosek** [reply]
That's good to hear :) at least you guys are still carrying the torch. Thanks for clarification.
